src/redux/thunks/utils.ts

```diff
@@ -48,6 +49,7 @@ export function createPreviewResult(resourcesYaml: string, previewResourceId: st
  */
 
 export function createPreviewRejection(thunkAPI: any, title: string, message: string) {
+  ipcRenderer.send('check-missing-dependency');
```

Comment:
Not sure from here! But I couldn't find a better place to trigger this event.

Reply:
I think it's enough for now to just check for these dependencies on startup - we can remove this check for now